### PR TITLE
 qolibri: 2019-07-22 -> 2.1.4

### PR DESCRIPTION
**Pull request by Jacavena**
###### Description of changes
* https://github.com/ludios/qolibri/releases/tag/2.1.4
* Install the desktop file in the source folder

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/qolibri/default.nix [hunk 1]
```diff
@@ -3,20 +3,24 @@
...
   pname = "qolibri";
-  version = "2019-07-22";
+  version = "2.1.4";
 
   src = fetchFromGitHub {
     owner = "ludios";
     repo = "qolibri";
-    rev = "b58f9838d39300cba444eba725a369181c5d746b";
-    sha256 = "0kcc6dvbcmq9y7hk8mp23pydiaqz6f0clg64d1f2y04ppphmah42";
+    rev = "c7f2a648d761b8367a00cae11977df2154fa630e";
```
Comment (sikmir) [inline pkgs/applications/misc/qolibri/default.nix:11]:
```suggestion
    rev = version;
```

File: pkgs/applications/misc/qolibri/default.nix [hunk 2]
```diff
@@ -3,20 +3,24 @@
 
 mkDerivation {
```
Comment (sikmir) [inline pkgs/applications/misc/qolibri/default.nix:4]:
```suggestion
stdenv.mkDerivation rec {
```

File: pkgs/applications/misc/qolibri/default.nix [hunk 3]
```diff
@@ -3,20 +3,24 @@
...
     repo = "qolibri";
-    rev = "b58f9838d39300cba444eba725a369181c5d746b";
-    sha256 = "0kcc6dvbcmq9y7hk8mp23pydiaqz6f0clg64d1f2y04ppphmah42";
+    rev = "c7f2a648d761b8367a00cae11977df2154fa630e";
+    sha256 = "jyLF1MKDVH0Lt8lw+O93b+LQ4J+s42O3hebthJk83hg=";
   };
 
   nativeBuildInputs = [ pkg-config cmake ];
   buildInputs = [
     libeb lzo qtbase qtmultimedia qttools qtwebengine
```
Comment (sikmir) [inline pkgs/applications/misc/qolibri/default.nix:17]:
```suggestion
    libeb lzo qtmultimedia qtwebengine
```

**Comment by Jacavena** [comment]
Is there anything I need to do to get this merged?

**Comment by Jacavena** [comment]
Is everything okay now or do I still need to do something?

**Comment by sikmir** [comment]
> Is everything okay now or do I still need to do something?

Yes, please squash commits into single one (`qolibri: 2019-07-22 -> 2.1.4`), so it can be merged.

**Comment by Jacavena** [comment]
Is it okay now?

**Pull request merged**